src/pages/landing/Landing.tsx

```diff
@@ -0,0 +1,181 @@
+import React, { useState } from "react";
+
+import { Container, Box, Typography, Button, TextField } from "@mui/material";
+
+import "./Landing.css";
+
+const ConnectSection = (props: { onSubmit: () => void }) => (
+  <Box className="flex flex-row-reverse flex-1">
+    <Box className="flex flex-col justify-center m-0 min-w-[50vw] bg-white">
+      <Box className="text-center">
+        <Typography variant="h3" className="font-bold mb-4 tracking-[.2em]">
+          Certifyr.io
```

Comment:
```suggestion
          Testing Tool
```
We got a red light on the naming so for now, we'll just use a generic name